services/dynamic/json-path.spec.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const chai = require('chai')
+const { expect } = chai
+const jsonPath = require('./json-path')
+
+chai.use(require('chai-as-promised'))
```

Comment:
`chai-as-promised` was removed in https://github.com/badges/shields/pull/2359, but it's great for testing rejected promises. I migrated several other tests to `chai-as-promised`. But not all of them, because `rejectedWith` can verify an error class and a message only.